R/zchunk_L2322.Fert.R

```diff
@@ -76,272 +76,447 @@ module_energy_L2322.Fert_DISABLED <- function(command, ...) {
-      # typical flags, but there are others--see `constants.R`
-      add_flags(FLAG_LONG_YEAR_FORM, FLAG_NO_XYEAR) ->
+      add_precursors("energy/A322.globaltech_shrwt") ->
       L2322.GlobalTechShrwt_Fert
 
-    tibble() %>%
-      add_title("descriptive title of data") %>%
-      add_units("units") %>%
-      add_comments("comments describing how data generated") %>%
-      add_comments("can be multiple lines") %>%
-      add_legacy_name("L2322.GlobalTechCoef_Fert") %>%
-      add_precursors("precursor1", "precursor2", "etc") %>%
-      # typical flags, but there are others--see `constants.R`
-      add_flags(FLAG_LONG_YEAR_FORM, FLAG_NO_XYEAR) ->
+    L2322.GlobalTechCoef_Fert %>%
+      add_title("Energy inputs and coefficients of global fertilizer energy use and feedstocks technologies") %>%
+      add_units("Unitless") %>%
+      add_comments("Interpolated orginal data into all model years") %>%
+      add_legacy_name("energy/A322.globaltech_coef") %>%
+      add_precursors("L1322.Fert_NEcost_75USDkgN_F") ->
       L2322.GlobalTechCoef_Fert
 
-    tibble() %>%
-      add_title("descriptive title of data") %>%
-      add_units("units") %>%
-      add_comments("comments describing how data generated") %>%
-      add_comments("can be multiple lines") %>%
+    L2322.GlobalTechCost_Fert %>%
+      add_title("Non-energy costs of global fertilizer manufacturing technologies") %>%
+      add_units("1975USDkgN") %>%
+      add_comments("Claculated cost from L1322.Fert_NEcost_75USDkgN_F") %>%
       add_legacy_name("L2322.GlobalTechCost_Fert") %>%
-      add_precursors("precursor1", "precursor2", "etc") %>%
-      # typical flags, but there are others--see `constants.R`
-      add_flags(FLAG_LONG_YEAR_FORM, FLAG_NO_XYEAR) ->
+      add_precursors("L1322.Fert_NEcost_75USDkgN_F") ->
       L2322.GlobalTechCost_Fert
 
-    tibble() %>%
-      add_title("descriptive title of data") %>%
-      add_units("units") %>%
-      add_comments("comments describing how data generated") %>%
-      add_comments("can be multiple lines") %>%
+    L2322.GlobalTechCapture_Fert %>%
+      add_title("CO2 capture fractions from global fertilizer production technologies with CCS") %>%
+      add_units("Unitless") %>%
+      add_comments("Interpolated orginal data into all model years") %>%
       add_legacy_name("L2322.GlobalTechCapture_Fert") %>%
-      add_precursors("precursor1", "precursor2", "etc") %>%
-      # typical flags, but there are others--see `constants.R`
-      add_flags(FLAG_LONG_YEAR_FORM, FLAG_NO_XYEAR) ->
+      add_precursors("energy/A322.globaltech_co2capture") ->
       L2322.GlobalTechCapture_Fert
 
-    tibble() %>%
-      add_title("descriptive title of data") %>%
-      add_units("units") %>%
-      add_comments("comments describing how data generated") %>%
-      add_comments("can be multiple lines") %>%
+    L2322.GlobalTechShutdown_Fert %>%
+      add_title("Global tech lifetime and shutdown rate") %>%
+      add_units("NA") %>%
```

Comment:
I think lifetime is in years

Reply:
Particularity for this one it was an empty tibble so no unit indicated. The tibble now has been removed from the chunk.